### ISSUE TITLE
Stance doesn't get saved into the database.

### ISSUE THREAD

**Issue by bbeckann** [renzuzu/renzu_stancer#29]
Hi @renzuzu,

I just installed this script today. I'm running ESX and oxmysql. So after giving me the stancekit I use it. Then I use the /stance command and edit the position of the car. After that I press save but when I /dv or park the vehicle. It returns to the car's default position. And when I try to do /stance again after pulling it out of the garage, it doesn't work. It says it is not installed in the car. But when I try to use the kit again, it gives the same error of not being able to install it in the car. There are no errors on the server console and none on the client console either. I've tried looking in the own_vehiclestable in the database, but I can't find anything to do with the posture. Where should this be saved? I did the same process with the spwan adm car, it is saving correctly  
![image](https://user-images.githubusercontent.com/95275692/223183299-c9493027-bbb7-4505-8afd-72992016c057.png)
![image](https://user-images.githubusercontent.com/95275692/223183552-87538eeb-0d24-4b09-91a9-4be4a49b5b29.png)

**Comment by renzuzu** [comment]
Hi bro thank for opening issue,

I will investigate this problem
It seems it works fine in me, 

Just to confirm before i analyze this

Your using one symc and lateat version of stancer right?

**Comment by bbeckann** [comment]
hi bro, thanks for the feedback.. yes i am using the latest version of all your scripts, it works fine when i use a car generated by adm spwan it saves correctly.. however with own vehicles it does not work does not save. I would be grateful if you would take a look.

**Comment by bbeckann** [comment]
![image](https://user-images.githubusercontent.com/95275692/223225380-1e0d68cb-b325-4638-ae26-c80f2bcaed83.png)

{DOLU} = THIS ADM SPWAN VEHICLE SAVING CORRECTLY
{MCZGW345} = VEHICLE PURCHASED IN STORE MODIFIED IN /STANCE NOT SAVED

**Comment by renzuzu** [comment]
the table your showing is for tuners.
the stancers saved in KVP in server.

i updated the stancer, you can try again and reproduce the problem. let me know

**Comment by bbeckann** [comment]
hello again I just updated the instancer , I bought the vehicle after I made the changes in /stancer . and kept it in the garage, when removing and placing
 /stancer again appears that the vehicle was not saved, even when I use it on the item again

![image](https://user-images.githubusercontent.com/95275692/223278470-268daa8f-dcbc-422b-b16c-8d6d881a8b72.png)

**Comment by renzuzu** [comment]
are you using old version of artifact?

try this new update
https://github.com/renzuzu/renzu_stancer/archive/refs/heads/main.zip

There should be a print message in f8 console when you use the stancer command
show me if this update still not working.

**Comment by bbeckann** [comment]
After downloading through your link I installed the update. I used the /stance command as shown in the image, it gave me this information in the console.
![image](https://user-images.githubusercontent.com/95275692/223292856-5e0ed971-e716-4fda-ad32-953dc49e3eaf.png)


soon afterwards I put the vehicle in the garage, soon after I removed it, as you can see in the image, the vehicle returned to normal settings. with this information from (nil) in the console
![image](https://user-images.githubusercontent.com/95275692/223293496-c0fcd1b6-134e-4a01-b6eb-7ec4997a3fec.png)

**Comment by renzuzu** [comment]
your using server vehicle spawn?

**Comment by bbeckann** [comment]
I'm not using it.. this vehicle above was purchased at ( esx_vehicleshop ) but I did a test with adm vehicle spwan

![image](https://user-images.githubusercontent.com/95275692/223294207-147db4ad-48c7-488b-81f5-bb0ad45c9f0b.png)
I did the same process on a spwan adm vehicle from the script (dolu_tool) that I use in my project... as shown in the image, it is saved correctly, I don't know how to explain it, but what you can see is that it is saving only the vehicles that are not your property.

shows in the image vehicle from spwan came saved
![image](https://user-images.githubusercontent.com/95275692/223294703-e29b4de8-3d45-4c6c-b879-56bd95e7fcb6.png)

As you can see in these two images below, they refer to the vehicle's license plate within the game. soon after the same card in the sql database

1
![image](https://user-images.githubusercontent.com/95275692/223294857-45755def-6b8a-4786-8952-d8f37da38c84.png)

2 sql 
![image](https://user-images.githubusercontent.com/95275692/223295095-048c7c1a-9f20-4987-8967-e8816d042100.png)

**Comment by renzuzu** [comment]
can you try normal vehicle with normal plate.. this could be conflict on PLATE, when plates is < 8 it will make a spaces
i dont know whats your garage and hows that setting vehicle plates.

i tried replicating your plate DOLU, it seems works fine in my end. as you said that your spawn vehicle from admin saved correctly. its using normal plates with max of 8 letters.

**Comment by bbeckann** [comment]
Build version of the game I'm using  = 5949

I'm using normal vehicles. purchased in esx_vehicleshop script but not saved

yes exactly but this board (DOLU) is generated by my adm spwan , I'll leave below some settings of my board, thanks in advance for helping me I found your script well prepared, I hope it works very well.



first my esx legacy setup

resources\[core]\es_extended/config.lua 
![image](https://user-images.githubusercontent.com/95275692/223300240-0c992fb4-b1ce-4cd9-84ad-1ea33bc2874a.png)
`Config.SpawnVehMaxUpgrades = false -- admin vehicles spawn with max vehcle settings
Config.CustomAIPlates = 'MCZ.A111' -- Custom plates for AI vehicles `

right below the configuration of the renzu_tuners cards (note, I don't know if there is any connection between the scripts that is more valid to show) I left the same configuration of my esx es_extended .

resources\[veiculos]\renzu_tuners/config.lua

![image](https://user-images.githubusercontent.com/95275692/223301491-9b97d489-1e67-44ff-9312-13368a961674.png)


`config.plateprefix = 'MCZ' -- recommended 3 char
config.plateformat = config.plateprefix..'A111' -- this will make plate CFXIE91F for example.. this is required to prevent local vehicles/ admin spawn entity filling up global state vehicle stats -- docs: https://docs.fivem.net/natives/?_0x79780FD2
`

**Comment by renzuzu** [comment]
you can find out game build by typing this in server console
```
sv_enforceGameBuild
```

**Comment by bbeckann** [comment]
![image](https://user-images.githubusercontent.com/95275692/223304550-309a388c-e80d-48de-b99e-57d769957546.png)

**Comment by bbeckann** [comment]
in that case how do i update ?

**Comment by renzuzu** [comment]
are you using esx_garage?

**Comment by renzuzu** [comment]
okay thats solve why the new update not working..
the only unsolve is why the statebag is not triggering from entityCreated Event from server?
only Server Setter vehicle spawn will cause this.

can you tell me how you spawn the vehicle? in garage ? what garage resource?

if this is the one your using
https://github.com/esx-framework/esx_garage/blob/ce82819fc097239211679d5bace3e52360981385/server/main.lua#L18

it will not work, that resource spawn the vehicle via server. the logic on this resource will not work.
i will update the server files to catch the statebag change to set stancer statebag properly.

i did not know that they really implement such a method. 🤦‍♂️

**Comment by bbeckann** [comment]
Yes I'm using esx_garage, I'm happy to be helping in some way to improve, I'm waiting for your stancer update, I suppose it's the same in nitro.. try to make an update that works universally, it would be interesting for you to create discord so that we can get in touch contact to report bug and help you in any way.

**Comment by renzuzu** [comment]
you can change the game build via server.cfg

```
set sv_enforceGameBuild 2699  
``` 
or if your using txadmin
go change it there too

**Comment by bbeckann** [comment]
![image](https://user-images.githubusercontent.com/95275692/223333331-17dfb448-ec4c-4229-8734-21b43aaf72b8.png)

hello bro in your update it worked super well, it's saving. but I made the modifications but when I took it out of the garage the third time my car gave 'dv' and went to impund.. I left the video of what is happening. and it gives me this error in the console

https://user-images.githubusercontent.com/95275692/223334302-822b2083-f034-40dd-ad69-9a34336752cf.mp4

**Comment by renzuzu** [comment]
i probably know whats causing this, when the server entity gets deleted the netid are re use again, specialy in your case if you are in testserver. i will push an update but im finishing some task adding more feat in stancer, like wheel size.

**Comment by bbeckann** [comment]
Hi Brother how are you? I hope you are well. I just tested the new update man, it's saving perfect, perfect! . but in this attempt I noticed an error in the console as shown in the image below

![image](https://user-images.githubusercontent.com/95275692/223456350-953e8b29-dec0-4144-b592-5e99d758dfee.png)

Another mistake, which I don't consider a mistake, as you said you were modifying the script to change the wheel size, I noticed that before it could rotate vertically both front and back, when I tried to modify I noticed that it only rotates the wheel as described in the video below .

https://user-images.githubusercontent.com/95275692/223457184-2d164d2b-f6be-4f2c-b02a-85a48175e7f6.mp4
